lib/PlaidClient.js

```diff
@@ -422,15 +422,16 @@ function(asset_report_token, cb) {
   }, cb);
 };
 
-// createPaymentRecipient(String, String, Object, Function)
+// createPaymentRecipient(String, String, Object, Object, Function)
 Client.prototype.createPaymentRecipient =
-  function(name, iban, address, cb) {
+  function(name, iban, address, bacs, cb) {
```

Comment:
What would the expected behavior be if both an IBAN and a BACS was provided?

Reply:
Right now it will use BACS over IBAN but will not throw a validation error.

Reply:
Is there a valid use case where both would be provided? If not maybe how come we aren't throwing a validation error?

Reply:
A client may want to use the same recipient for domestic and international payments

Reply:
Ahh yea ok I just saw the thread as well